src/lib/metricsMgr/MetricsManager.cpp

```diff
@@ -142,8 +142,26 @@ void MetricsManager::add(const std::string& srv, const std::string& subServ, con
     subService = &subService[1];
   }
 
+  //
+  // Convert tenant/service to lowercase
```

Comment:
Moreover... maybe the lower-casing should be done at the very beginning (just after reading it from the HTTP header) so the lowercase version would be used in all the logic (both metric related code and "regular" processing)?